# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rapidsvn/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur rapidsvn-0.9.1-orig/src/action.hpp rapidsvn-0.9.1/src/action.hpp
--- rapidsvn-0.9.1-orig/src/action.hpp	2006-02-14 14:30:20.000000000 +0000
+++ rapidsvn-0.9.1/src/action.hpp	2006-05-20 10:57:41.348103000 +0000
@@ -300,7 +300,7 @@
     * @return temporary filename
     */
   svn::Path
-  Action::GetPathAsTempFile (
+  GetPathAsTempFile (
     const svn::Path & path,
     const svn::Revision & revision = svn::Revision::HEAD);
 
diff -Nur rapidsvn-0.9.1-orig/src/preferences_dlg.cpp rapidsvn-0.9.1/src/preferences_dlg.cpp
--- rapidsvn-0.9.1-orig/src/preferences_dlg.cpp	2006-02-14 14:30:20.000000000 +0000
+++ rapidsvn-0.9.1/src/preferences_dlg.cpp	2006-05-20 11:04:59.383478500 +0000
@@ -95,7 +95,7 @@
 {
 public:
 
-  ProgramsPanel::ProgramsPanel (wxWindow* parent, Preferences * prefs)
+  ProgramsPanel (wxWindow* parent, Preferences * prefs)
     : wxPanel (parent), m_prefs (prefs)
   {
     InitializeData ();
diff -Nur rapidsvn-0.9.1-orig/src/rapidsvn_frame.hpp rapidsvn-0.9.1/src/rapidsvn_frame.hpp
--- rapidsvn-0.9.1-orig/src/rapidsvn_frame.hpp	2006-02-14 14:30:20.000000000 +0000
+++ rapidsvn-0.9.1/src/rapidsvn_frame.hpp	2006-05-20 11:01:54.995955000 +0000
@@ -161,7 +161,7 @@
 
   // Enable/disable action menu items
   bool
-  RapidSvnFrame::ValidateIDActionFlags (int id, unsigned int selectionActionFlags);
+  ValidateIDActionFlags (int id, unsigned int selectionActionFlags);
   void OnUpdateCommand (wxUpdateUIEvent & updateUIEvent);
 
   const svn::Targets
